### DISCUSSION TITLE
Moving an innernet-server

### DISCUSSION THREAD

**Discussion by jimt** [tonarino/innernet#217] [Q&A] [answered]
The peers appear to know the server by its IP address rather than a hostname.  Is it possible to move the server of an established network?

**Comment by strohel** [comment]
Hi @jimt, I _think_ it should suffice to edit `[server]` -> `external-endpoint` value in `/etc/innernet/<network-name>.conf` on all clients. But I haven't tested that.

**Comment by jimt** [comment, marked as answer]
Yes, I feared it would require scripting something to visit all the nodes.  Thanks.

**Reply by mcginty** [reply]
Yep, there's unfortunately no quick way to do that when the server is only known by IP. In the future, it's much simpler to migrate if you specify a hostname as your server.

I suppose there could be a feature where the server (if it's still contactable) can tell peers to use a certain hostname instead of an IP, but that seems a little convoluted :).
